tasks/components/modals/ConfirmDialog.tsx

```diff
@@ -12,6 +12,12 @@ type ConfirmDialogTranslation = {
   decline: string
 }
 
+export enum ConfirmDialogType{
+  Positive = 'Positive',
+  Neutral = 'Neutral',
+  Negative = 'Negative'
+}
```

Comment:
Rather use a string union than an actual enum
```suggestion
export type ConfirmDialogType = 'positive' | 'negative' | 'neutral'
```